src/services/models/authModels.js

```diff
@@ -29,13 +29,14 @@ export const permissions = {
   createPackages: 'CreatePackages',
 
   createDeployments: 'CreateDeployments',
-  delteDeployments: 'DeleteDeployments'
+  deleteDeployments: 'DeleteDeployments'
```

Comment:
This reference wasn't used any where else? Looks like this fixes a bug in perms :)

Reply:
It wasn't used anywhere (yet), but it was misspelled... so I fixed it.